manual-tests.md

```diff
@@ -76,6 +76,15 @@
 5. Run `go-hello`.
 
 
+# Test cross-compilation with Rust
+
+1. Go to integration_tests/snaps/rust-hello.
+2. Run `snapcraft snap --target-arch=armhf`.
+3. Copy the snap to a Raspberry Pi.
```

Comment:
I was talking about this with @elopio a couple of weeks ago. We could consider an integration test using qemu. But we can't run the test snap on a native armhf machine. So the manual test is specifically for this.